basil/HL/sensirion_ekh4.py

```diff
@@ -76,7 +80,11 @@ def get_humidity(self, channel=None, min_val=0, max_val=100):
             if ret[2 + 2 * j] == "7f" and ret[2 + 2 * j + 1] == "ff":
                 values.append(None)
             else:
-                values.append(self.cal_ret(ret[2 + 2 * j] + ret[2 + 2 * j + 1]))
+                val = self.cal_ret(ret[2 + 2 * j] + ret[2 + 2 * j + 1])
+                if val < min_val or val > max_val:
```

Comment:
So, you would like to re-implement the feature of checking the return value, that you have deleted before?

Reply:
it was never implemented, only the parameters were there but no check. i have a cleaner workaround now, will push tonight

Reply:
I implemented it in my original version 5 years ago. But you did not delete it, [Toko did](https://github.com/SiLab-Bonn/basil/commit/fa4f9f82fe6ce12a2a744b5fcc875ddc36e8d07f).

Reply:
some error correction and more secure handling of corrupt data now implemented, so this workaround was removed completely